src/components/components/ebay-tooltip-overlay/template.marko

```diff
@@ -0,0 +1,35 @@
+<div
+    class="${data.type}__overlay"
+    role=(data.type === 'tourtip'?'region':'tooltip')
```

Comment:
Only classic tooltip should have role of tooltip, i.e. not the Infotip.

Reply:
Cool. Easy change.

Reply:
So ... what role should the infotip have?

Reply:
No role.